src/vochain/vochain.proto

```diff
@@ -170,8 +170,11 @@ message Process {
 	CensusOrigin censusOrigin = 20;
 	ProcessResult results = 21;
 	repeated bytes resultsSignatures = 22;
-
 	optional uint32 ethIndexSlot = 23;
+	// SourceBlockHeight is the block height required for on-chain census
```

Comment:
Let's make it more generic: `// SourceBlockHeight is the block height of the origin blockchain (if any)`
Same for `owner`